src/main/java/uk/gov/justice/hmpps/casenotes/notes/internal/Type.kt

```diff
@@ -0,0 +1,73 @@
+package uk.gov.justice.hmpps.casenotes.notes.internal
+
+import jakarta.persistence.Column
+import jakarta.persistence.Entity
+import jakarta.persistence.EntityListeners
+import jakarta.persistence.GeneratedValue
+import jakarta.persistence.GenerationType
+import jakarta.persistence.Id
+import jakarta.persistence.JoinColumn
+import jakarta.persistence.ManyToOne
+import jakarta.persistence.Table
+import org.hibernate.annotations.Immutable
+import org.springframework.data.jpa.repository.EntityGraph
+import org.springframework.data.jpa.repository.JpaRepository
+import uk.gov.justice.hmpps.casenotes.audit.AuditedEntityListener
+import uk.gov.justice.hmpps.casenotes.audit.SimpleAudited
+
+@Immutable
+@Entity
+@Table(name = "case_note_type")
+@EntityListeners(AuditedEntityListener::class)
+class Type(
```

Comment:
I think we have a few different names for these now. It's type and parent type for the get types path and type and category here. They're types and sub types on the UI and that's how we describe them when discussing with the business although I do agree that category and type is closest. Worth a discussion